llvm/lib/Target/AArch64/AArch64.td

```diff
@@ -684,7 +684,7 @@ def HasV8_9aOps : SubtargetFeature<
 
 def HasV9_0aOps : SubtargetFeature<
   "v9a", "HasV9_0aOps", "true", "Support ARM v9a instructions",
-  [HasV8_5aOps, FeatureMEC, FeatureSVE2]>;
+  [HasV8_5aOps, FeatureSVE2]>;
```

Comment:
This looks like it affects every CPU that declares support for armv9; is this correct for all of them?

Reply:
I will update this patch to leave FEAT_MEC unchanged for now, but we will probably (in future) disable it for all CPUs (and add a flag to enable), since it's a v9.2a and higher optional feature.